### PR TITLE
Creating jobpost service (still on working/progress)

### PR DESCRIPTION
**Pull request by dilippoudel**

### CODE REVIEW

File: src/pages/EditProfile/index.tsx [hunk 1]
```diff
@@ -8,70 +8,82 @@ import userImg from '../../media/user-img.svg'
...
-          </div>
-          <div className="m-auto hide-sm">
-            <img src={userImg} alt="userImg-svg" />
-            <br />
-            <CustomButton text="Upload" className="my-2 py-2" />
-          </div>
-        </div>
         <Form.Label as="legend" column sm={2} className="ml-n3">
           Education
         </Form.Label>
```
Comment (Nejo12) [inline src/pages/EditProfile/index.tsx]: I think you did not accept the incoming changes in this file and This cleared everything I worked on today.  Kindly check if you can revert this file.

File: src/scss/_mobile.scss [hunk 2]
```diff
@@ -9,10 +9,6 @@
     display: none !important;
   }
 
-  .personal-form {
-    width: 100% !important;
-  }
-
```
Comment (Nejo12) [inline src/scss/_mobile.scss]: Same here as well.

File: src/scss/styles.scss [hunk 3]
```diff
@@ -62,11 +53,9 @@ a:hover {
   text-decoration: none;
 }
 
-// job seeker profile
-.personal-form {
-  width: 60%;
+.user-img {
+  right: 20%;
 }
-
```
Comment (Nejo12) [inline src/scss/styles.scss]: Here too... :(

**Review by Nejo12 — CHANGES_REQUESTED** [review]
Could you check and correct if the reverts can be done before merging?

**Comment by dilippoudel** [comment]
out dated

**Pull request closed without merging**